ensembl/htdocs/info/genome/variation/species/species_detailed_counts.html

```diff
@@ -1405,14 +1405,14 @@ <h2 id="prediction" style="margin-top:40px">Prediction data</h2><p style="paddin
             </a>
            </div>
            <div style="float:left;margin-left:6px;padding-top:2px">
-             <div class="bigtext">Chicken</div>
+             <div class="bigtext">Chicken (maternal Broiler)</div>
              <div class="small" style="font-style:italic">Gallus gallus</div>
            </div>
            <div style="clear:both"></div>
          </div>
       </td>
-      <td style="text-align:right"><span class="vdoc_var_count vdoc_thousand" title="Over 229,000 variants with sift data">229 K</span></td>
-      <td style="text-align:right">-</td>
+      <td style="text-align:right"><span class="vdoc_var_count vdoc_thousand" title="Over 256,000 variants with sift data">256 K</span></td>
```

Comment:
As shown afterwards, I think it should be better to use `SIFT predictions` instead:

```suggestion
      <td style="text-align:right"><span class="vdoc_var_count vdoc_thousand" title="Over 256,000 variants with SIFT predictions">256 K</span></td>
```